lib/faraday.rb

```diff
@@ -37,33 +37,32 @@ class << self
     # Public: Tells faraday to ignore the environment proxy (http_proxy).
     attr_accessor :ignore_env_proxy
 
-    # Public: Initializes a new Faraday::Connection.
+    # Initializes a new {Connection}.
     #
-    # url     - The optional String base URL to use as a prefix for all
-    #           requests.  Can also be the options Hash.
-    # options - The optional Hash used to configure this Faraday::Connection.
-    #           Any of these values will be set on every request made, unless
-    #           overridden for a specific request.
-    #           :url     - String base URL.
-    #           :params  - Hash of URI query unencoded key/value pairs.
-    #           :headers - Hash of unencoded HTTP header key/value pairs.
-    #           :request - Hash of request options.
-    #           :ssl     - Hash of SSL options.
-    #           :proxy   - Hash of Proxy options.
+    # @param url [String,Hash] The optional String base URL to use as a prefix for all
+    #           requests.  Can also be the options Hash. Any of these values
+    #           will be set on every request made, unless overridden for a
+    #           specific request.
+    # @param options [Hash]
+    # @option options [String] :url Base URL
+    # @option options [Hash] :params Hash of URI query unencoded key/value pairs.
+    # @option options [Hash] :headers Hash of unencoded HTTP header key/value pairs.
+    # @option options [Hash] :request Hash of request options.
+    # @option options [Hash] :ssl Hash of SSL options.
+    # @option options [Hash] :proxy Hash of Proxy options.
+    # @return [Faraday::Connection]
     #
-    # Examples
+    # @example
     #
     #   Faraday.new 'http://faraday.com'
-    #
-    #   # http://faraday.com?page=1
+    #   # => http://faraday.com?page=1
+    #   
     #   Faraday.new 'http://faraday.com', :params => {:page => 1}
-    #
-    #   # same
+    #   # => http://faraday.com?page=1
     #
     #   Faraday.new :url => 'http://faraday.com',
-    #     :params => {:page => 1}
-    #
-    # Returns a Faraday::Connection.
+    #               :params => {:page => 1}
+    #   # => Faraday::Connection
```

Comment:
This just demonstrates how the old documentation was confusing.
I think results were put on top of the corresponding example, not below.
So the correct example -> results would be:
```ruby
Faraday.new 'http://faraday.com'
# => Faraday::Connection to http://faraday.com

Faraday.new 'http://faraday.com', :params => {:page => 1}
# => Faraday::Connection to http://faraday.com?page=1

Faraday.new :url => 'http://faraday.com',
            :params => {:page => 1}
# => Faraday::Connection to http://faraday.com?page=1
```

Reply:
Thanks for good review item!

Reply:
Using `@example` more than once, and using its title feature helps this method, I think:

![image](https://user-images.githubusercontent.com/211/35187310-2c59dfb8-fe22-11e7-94df-6ad075591f05.png)

Reply:
That's looks even better 👍 !